### DISCUSSION TITLE
Are you aware of imag-pim

### DISCUSSION THREAD

**Discussion by ibotty** [pimalaya/himalaya#64] [Show and tell]
https://imag-pim.org/

There might be some overlap.  Maybe your enthusiasm can spark some activity in imag as well.

**Comment by soywod** [comment]
The project does not seem to be so active, but it's exactly where I would like to tend (a collection of PIM tools for the terminal). I will investigate. Thanks for the sharing!

**Comment by soywod** [comment]
The domain looks dead as I am writing this comment in 2026.
